wasm/lib/src/convert.rs

```diff
@@ -125,12 +125,10 @@ pub fn py_to_js(vm: &VirtualMachine, py_obj: PyObjectRef) -> JsValue {
         || objtype::isinstance(&py_obj, &vm.ctx.bytearray_type())
     {
         let bytes = objbytes::get_value(&py_obj);
-        let arr = Uint8Array::new_with_length(bytes.len() as u32);
-        for (i, byte) in bytes.iter().enumerate() {
-            Reflect::set(&arr, &(i as u32).into(), &(*byte).into())
-                .expect("setting Uint8Array value failed");
+        unsafe {
```

Comment:
What's this `unsafe` needed for?

Reply:
`Uint8Array::view` is an `unsafe fn` because it provides a direct view into the WASM linear memory; if you were to allocate something with Rust that view would probably become invalid. It's safe because I then copy the array using `Uint8Array::slice`.

Reply:
Okay, could you add that in a comment?

Reply:
Done.

Reply:
(Also, how is this related to cargo dependencies?)

Reply:
Oh, its a feature only in more recent js-sys versions, and when I tried to update a few weeks ago everything broke. I figured it'd be a bot more involved getting everything to work, but I guess not:smile:

Reply:
Specifically `Uint8Array::{view,copy_to}`